.semaphore/semaphore.yml

```diff
@@ -34,10 +35,15 @@ blocks:
         - name: "Install"
           commands:
             - cache restore
+            - tar zcvf cache.tgz .m2 target
+            - artifact push project cache.tgz
+            - rm ~/.m2/settings.xml
```

Comment:
why are we removing the settings here?

Reply:
this was a test to understand more about the eventual need to migrate to non-Confluent infrastructure over time as well as understand why the initial cache removal was failing to resolve all artifacts.

Reply:
this project should be able to build without any internal dependencies, so no maven settings file should be necessary unless we want to rely on our own maven cache instead of pulling from maven central.

Reply:
so think it's fine to remove the .m2 settings file, but let's add a comment to explain why we are doing this so we know when to remove it.

Reply:
sure. Will revert this change for now, we can track the change to using the public maven central in another phase of migration.

Reply:
I'm fine removing it as long as the build passes and we add a comment to explain why.